src/Equinor.ProCoSys.BusSender.Infrastructure/Repositories/EventRepository.cs

```
@@ -41,6 +42,11 @@ public EventRepository(BusSenderServiceContext context, ILogger<EventRepository>
 
             return events.Single();
         }
+        catch (Exception ex)
```

Why do we have to "swallow" exception now, since we didn't have to before this PR?